### PR TITLE
Error Code 0x80244018 was missing

### PR DESCRIPTION
**Pull request by joinimran**
I have added the related information.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2343

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/upgrade/resolution-procedures.md [hunk 1]
```diff
@@ -534,6 +534,12 @@ Download and run the media creation tool. See [Download windows 10](https://www.
 </td>
 </tr>
 <tr>
+<td>0x80244018</td>
+<td>Your machine is connected through a proxy server.</td>
+<td>Make sure Automatically Detect Settings is checked in internet options. (Control Panel - Internet Options - Connections - LAN Settings)
```
Comment (nenonix) [inline windows/deployment/upgrade/resolution-procedures.md]:
```suggestion
<td>Make sure Automatically Detect Settings is selected in internet options. (Control Panel > Internet Options > Connections > LAN Settings)
```

**Comment by mypil** [comment]
@joinimran - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by joinimran** [comment]
@mypil Ready to merge.

**Comment by mypil** [comment]
@greg-lindsay - Can you please assist to check, approve and merge the content of this PR based on issue #2343?

Thank you.

**Comment by mypil** [comment]
@lomayor  @greg-lindsay @AndreaBarr  - We would like to follow up. Can you please assist to check, approve and merge the content of this PR based on issue #2343?

Thank you.

**Pull request merged**